### PR TITLE
Homepage design update

### PR DESCRIPTION
**Pull request by Shan-Weaviate**
### What's being changed:

- [ ] **Homepage** components update/created as per new design. This has updated the homepage layout in index.jsx
- [ ] **Homepage** global css (custom.scss) change to accommodate new design

### Type of change:

- [ ] **Website** updates (non-breaking change to update main page, company pages, pricing, etc)

### How Has This Been Tested?

- [ ] **Github action** – automated build completed without errors
- [ ] **Local build** - the site works as expected when running `yarn start`

> note, you can run `yarn verify-links` to test site links locally

### CODE REVIEW

File: src/components/Home/IntegrationsUpdate/index.jsx [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+          <div className={styles.right}>
+            <h2>Integrations</h2>
+            <p>
+              Besides Weaviate's capabilities to bring your own vectors, you can
+              also choose one of Weaviate's modules with out-of-the-box support
+              for vectorization. You can also pick from a wide variety of
+              well-known neural search frameworks with Weaviate integrations.
+            </p>
+            {/*    <ButtonContainer position='left'>
+            <LinkButton link="/developers/weaviate/modules" newTab={false}>Learn about Weaviate modules</LinkButton>
```
Comment (Weroiko) [inline src/components/Home/IntegrationsUpdate/index.jsx:21]: if not needed, remove it, please

File: src/components/Home/JoinCommunityUpdate/index.jsx [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+            <p className={styles.rightText}>
+              Stay updated and subscribe to our newsletter
+            </p>
+
+            <div className={styles.communityForm}>
+              {/*    <form className={styles.communityForm}>
+              <input
+                className={styles.subscribeInput}
+                type="field"
```
Comment (Weroiko) [inline src/components/Home/JoinCommunityUpdate/index.jsx:65]: same here, if not needed, remove it, please

File: src/components/Home/WhatYouCanDoUpdate/index.jsx [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+              <div className={styles.codeContainer}>
+                <p className={styles.tabsText}>
+                  Perform lightning-fast pure vector similarity<br></br>search
+                  over raw vectors or data objects,<br></br> even with filters.
+                </p>
+                <CodeSnippet language="typescript" code={codeExample} />
+                {/*  <p className={styles.codeImage + ' ' + styles.code1}></p>
+                  <ButtonContainer position='right'>
+                  <LinkButton
+                    link="https://console.weaviate.io/console/query#weaviate_uri=https://demo.dataset.playground.semi.technology&graphql_query=%7B%0A%20%20Get%20%7B%0A%20%20%20%20Publication(%0A%20%20%20%20%20%20nearText%3A%20%7B%0A%20%20%20%20%20%20%20%20concepts%3A%20%5B%22fashion%22%5D%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20limit%3A%201%0A%20%20%20%20)%20%7B%0A%20%20%20%20%20%20name%0A%20%20%20%20%20%20_additional%20%7B%0A%20%20%20%20%20%20%20%20certainty%0A%20%20%20%20%20%20%20%20distance%0A%20%20%20%20%20%20%20%20vector%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D"
```
Comment (Weroiko) [inline src/components/Home/WhatYouCanDoUpdate/index.jsx:87]: to be removed?

File: src/components/Home/WhatYouCanDoUpdate/index.jsx [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+              <div className={styles.codeContainer}>
+                <p className={styles.tabsText}>
+                  Combine keyword-based search with vector search techniques for
+                  state-of-the-art results.
+                </p>
+                <CodeSnippet language="typescript" code={codeExample2} />
+                {/*  <p className={styles.codeImage + ' ' + styles.code2}></p>
+                  <ButtonContainer position='right'>
+                  <LinkButton
+                    link="https://console.weaviate.io/console/query#weaviate_uri=https://demo.dataset.playground.semi.technology&graphql_query=%7B%0A%20%20Get%20%7B%0A%20%20%20%20Article(%0A%20%20%20%20%20%20hybrid%3A%20%7B%0A%20%20%20%20%20%20%20%20query%3A%20%22Board%20games%20people%20are%20looking%20out%20for%22%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20limit%3A%2010%0A%20%20%20%20)%20%7B%0A%20%20%20%20%20%20summary%0A%20%20%20%20%20%20title%0A%20%20%20%20%20%20url%0A%20%20%20%20%20%20wordCount%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D"
```
Comment (Weroiko) [inline src/components/Home/WhatYouCanDoUpdate/index.jsx:102]: same here

File: src/components/Home/WhatYouCanDoUpdate/index.jsx [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+                <p className={styles.tabsText}>
+                  Use any generative model in combination with your data, for
+                  example to do Q&A over your dataset.
+                </p>
+                <CodeSnippet language="typescript" code={codeExample3} />
+                {/*  <p className={styles.codeImage + ' ' + styles.code3}></p>
+                <ButtonContainer position='right'>
+                  <LinkButton
+                    link="https://console.weaviate.io/console/query#weaviate_uri=https://demo.dataset.playground.semi.technology&graphql_query=%7B%0A%20%20Get%20%7B%0A%20%20%20%20Article(%0A%20%20%20%20%20%20ask%3A%20%7B%0A%20%20%20%20%20%20%20%20question%3A%20%22What%20movie%20did%20Ravensburger%20create%20a%20board%20game%20about%3F%22%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20limit%3A%201%0A%20%20%20%20)%20%7B%0A%20%20%20%20%20%20summary%0A%20%20%20%20%20%20title%0A%20%20%20%20%20%20url%0A%20%20%20%20%20%20wordCount%0A%20%20%20%20%20%20_additional%20%7B%0A%20%20%20%20%20%20%20%20answer%20%7B%0A%20%20%20%20%20%20%20%20%20%20result%0A%20%20%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D"
```
Comment (Weroiko) [inline src/components/Home/WhatYouCanDoUpdate/index.jsx:117]: same here

File: src/components/Home/WhatYouCanDoUpdate/styles.module.scss [hunk 6]
```diff
@@ -0,0 +1,171 @@
...
+  .right {
+    display: flex;
+    flex: 1;
+
+
+
+
+    div[data-state='active'] {
+      display: flex;
+      // flex-direction: column;
```
Comment (Weroiko) [inline src/components/Home/WhatYouCanDoUpdate/styles.module.scss:117]: to be removed?

File: src/css/custom.scss [hunk 7]
```diff
@@ -374,21 +448,22 @@ button.close {
...
+  .custom-page.homepage {
+    background-image: none;
+    background-color: #F5F5F5;
+}
+
   .hero__logo {
     background-image: url('/img/site/weaviate-logo-light.png');
   }
   .navbar {
     // background: transparent;
```
Comment (Weroiko) [inline src/css/custom.scss:465]: remove it?

**Review by Weroiko — CHANGES_REQUESTED** [review]

File: src/components/Home/IntegrationsUpdate/index.jsx (hunk 1 above)
Comment (Shan-Weaviate) [reply src/components/Home/IntegrationsUpdate/index.jsx]: Updated. Thanks :)

File: src/components/Home/WhatYouCanDoUpdate/index.jsx (hunk 3 above)
Comment (Shan-Weaviate) [reply src/components/Home/WhatYouCanDoUpdate/index.jsx]: Removed.

File: src/components/Home/WhatYouCanDoUpdate/index.jsx (hunk 4 above)
Comment (Shan-Weaviate) [reply src/components/Home/WhatYouCanDoUpdate/index.jsx]: Removed.

File: src/components/Home/WhatYouCanDoUpdate/styles.module.scss (hunk 6 above)
Comment (Shan-Weaviate) [reply src/components/Home/WhatYouCanDoUpdate/styles.module.scss]: Removed

**Review by Shan-Weaviate — COMMENTED** [review]
Review changes updated. Thanks

**Pull request merged**